src/otx/recipe/_base_/test.yaml

```diff
@@ -0,0 +1,26 @@
+callbacks:
```

Comment:
It seems that `_base_/test` is not proper naming since it only contains the callbacks, logger.

In my opinion, the base yaml of `train` or `test` should include more close contents about the `train` or `test`. So, we might add `accerlator` or `dist_config` or such other things..
i.e. https://github.com/open-mmlab/mmdetection/blob/main/configs/_base_/default_runtime.py

Reply:
`train.yaml` and `test.yaml` are the default configuration files that correspond to the arguments to `engine.train()` and `engine.test()`, respectively.
These files correspond to `configs/train.yaml` & `configs/test.yaml` in the current v2.

For `test.yaml`, I'm thinking that things that could go in kwargs rather than arguments are declared in the file, which could be confusing. I'll think about this a bit more.